src/test/java/org/apache/drillui/test/framework/testng/secure/login/LoginTest.java

```diff
@@ -16,23 +16,90 @@
  */
 package org.apache.drillui.test.framework.testng.secure.login;
 
+import org.apache.drillui.test.framework.initial.PropertiesConst;
 import org.apache.drillui.test.framework.initial.TestProperties;
+import org.apache.drillui.test.framework.initial.WebBrowser;
+import org.apache.drillui.test.framework.pages.BasePage;
+import org.apache.drillui.test.framework.pages.QueryResultsPage;
+import org.apache.drillui.test.framework.steps.webui.*;
 import org.apache.drillui.test.framework.testng.secure.BaseSecureTest;
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
 import org.testng.annotations.Test;
-import org.apache.drillui.test.framework.steps.webui.AuthSteps;
+
+import java.util.List;
+import java.util.Set;
 
 import static org.testng.Assert.*;
 
 public class LoginTest extends BaseSecureTest {
   @Test(groups = {"functional"})
   public void testLogin() {
     assertEquals(
-        AuthSteps.login(TestProperties.get("ADMIN_USER1_NAME"), TestProperties.get("ADMIN_USER1_PASSWORD")).getLogoutText(),
-        "Log Out (" + TestProperties.get("ADMIN_USER1_NAME") + ")", "Login failed");
+       new AuthSteps().login(PropertiesConst.ADMIN_1_NAME, PropertiesConst.ADMIN_1_PASSWORD).getLogoutText(),
+        "Log Out (" + PropertiesConst.ADMIN_1_NAME + ")", "Login failed");
   }
 
   @Test(groups = {"functional"})
   public void testLogout() {
-    assertEquals(AuthSteps.logOut().getLoginText(), "Log In", "Logout failed");
+    assertEquals(  new AuthSteps().logOut().getLoginText(), "Log In", "Logout failed");
+  }
+
+  @Test(groups = {"functional"})
+  public void AdminsViewVsUsersViewInSecurityMode()  {
+    //Step 1. Run query for Admin1
+    Assert.assertEquals("COMPLETED",new AuthSteps().login(PropertiesConst.ADMIN_1_NAME,PropertiesConst.ADMIN_1_PASSWORD)
```

Comment:
You can use method import here. There is no reason to write class name before every assert.

Reply:
Done, thanks